components/team_sidebar/team_sidebar_controller.jsx

```diff
@@ -145,7 +145,7 @@ export default class TeamSidebar extends React.Component {
                 );
             });
 
-        if (moreTeams) {
+        if (moreTeams && !global.mm_config.ExperimentalPrimaryTeam) {
```

Comment:
shouldn't this be something like `(!global.mm_config.ExperimentalPrimaryTeam || global.mm_config.ExperimentalPrimaryTeam === 'false')` ??

Reply:
@enahum the value is not boolean. It's either the team name or an empty string.